seleniumByTechnocredits/src/seleniumPractise/amarja/ExcelsheetModular.java

```diff
@@ -0,0 +1,84 @@
+package seleniumPractise.amarja;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import org.apache.poi.ss.usermodel.CellType;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+public class ExcelsheetModular {
+	Workbook myworkbook=null;
+	public ExcelsheetModular(String path) throws IOException
+	{
+		File file =new File(path);
+		FileInputStream input=new FileInputStream(file);
+		myworkbook=new XSSFWorkbook(input);
+	}
+	public String[][] getRow(String sheetName,int rowIndex)
+	{
+		Sheet sheet=myworkbook.getSheet(sheetName);
+		Row row=sheet.getRow(rowIndex);
+		int start=row.getFirstCellNum();
+		int end=row.getLastCellNum();
+		String data[] [] =new String [1][end];
+		for(int i=start;i<end;i++)
+		{
+			data[0][i]=(row.getCell(i).getStringCellValue());
+		}
+		return data;
+	}
+	
+public Object [][] getAllrows(String sheetName,boolean isheader)
+{
+	Sheet sheet=myworkbook.getSheet(sheetName);
+	int count=0;
+	int firstrow = sheet.getFirstRowNum();
+	System.out.println("Firstrow"+firstrow);
+	int lastrow= sheet.getLastRowNum();
+	System.out.println("Lastrow"+lastrow);
+	int firstcol = sheet.getRow(firstrow).getFirstCellNum();
+	int lastcol=sheet.getRow(0).getLastCellNum();
+	System.out.println("Lastcol"+lastcol);
+	Object[][] data =new Object [lastrow][lastcol];
+	System.out.println("rownumber" + firstrow);
+	if (isheader) {
+		firstrow++;
+		 count=1;
```

Comment:
count =1 is not required here as you have already declared the variable count at the start of method. If header is present it will increase count of firstrow by 1.